### PR TITLE
[FLINK-25800][docs] Update wrong links in the datastream/execution_mode.md page.

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

Update wrong links in the datastream/execution_mode.md page.

## Brief change log

Update wrong links in the datastream/execution_mode.md page.

## Verifying this change

Update wrong links in the datastream/execution_mode.md page.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
@flinkbot run azure

**Comment by RocMarshal** [comment]
Hi, @MartijnVisser Would you mind helping me to check it ? Thanks a lot.

File: docs/content.zh/docs/dev/datastream/execution_mode.md [hunk 1]
```diff
@@ -201,11 +201,11 @@ Checkpointing 用于故障恢复的特点之一是，在发生故障时，Flink
 
 `批`模式下的行为变化：
 
-* “滚动"操作，如 [reduce()]({{< ref "docs/dev/datastream/operators/overview" >}}#reduce) 或 [sum()]({{< ref "docs/dev/datastream/operators/overview" >}}#aggregations)，会对`流`模式下每一条新记录发出增量更新。在`批`模式下，这些操作不是"滚动”。它们只发出最终结果。
+* “滚动"操作，如 [reduce()]({{< ref "docs/dev/datastream/operators/overview" >}}#reduce) 或 sum()，会对`流`模式下每一条新记录发出增量更新。在`批`模式下，这些操作不是"滚动”。它们只发出最终结果。
```
Comment (wuchong) [inline docs/content.zh/docs/dev/datastream/execution_mode.md:204]: Why remove the `sum()` link?

Comment (RocMarshal) [reply docs/content.zh/docs/dev/datastream/execution_mode.md:204]: Glad to get your review .  @wuchong 

There isn't  any aggregate function section title in the docs/dev/datastream/operators/overview page, but we could make it mapping to the page   [flink/docs/content.zh/docs/dev/datastream/operators/windows.md](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/operators/windows/#aggregatefunction') .  

Please let me know what's your opinion.

**Comment by RocMarshal** [comment]
Hi, @MartijnVisser Could you help me to check it ? Thanks a lot.

File: docs/content.zh/docs/dev/datastream/execution_mode.md [hunk 2]
```diff
@@ -170,7 +170,7 @@ TaskManagers 将至少在下游任务开始消费它们前保留中间结果（
 
 ### 事件时间/水印
 
-在支持[事件时间]({{< ref "docs/dev/datastream/event-time/generating_watermarks" >}})方面，Flink 的流运行时间建立在一个事件可能是乱序到来的悲观假设上的，即一个时间戳 `t` 的事件可能会在一个时间戳 `t+1` 的事件之后出现。因为如此，系统永远无法确定在给定的时间戳 `T` 下，未来不会再有时间戳 `t < T` 的元素出现。为了摊平这种失序性对最终结果的影响，同时使系统实用，在`流`模式下，Flink 使用了一种名为 [Watermarks]({{< ref "docs/concepts/time" >}}#event-time-and-watermarks) 的启发式方法。一个带有时间戳 `T` 的水印标志着再没有时间戳 `t < T` 的元素跟进。
+在支持[事件时间]({{< ref "docs/concepts/time" >}}#notions-of-time-event-time-and-processing-time)方面，Flink 的流运行时间建立在一个事件可能是乱序到来的悲观假设上的，即一个时间戳 `t` 的事件可能会在一个时间戳 `t+1` 的事件之后出现。因为如此，系统永远无法确定在给定的时间戳 `T` 下，未来不会再有时间戳 `t < T` 的元素出现。为了摊平这种失序性对最终结果的影响，同时使系统实用，在`流`模式下，Flink 使用了一种名为 [Watermarks]({{< ref "docs/concepts/time" >}}#event-time-and-watermarks) 的启发式方法。一个带有时间戳 `T` 的水印标志着再没有时间戳 `t < T` 的元素跟进。
```
Comment (MartijnVisser) [inline docs/content.zh/docs/dev/datastream/execution_mode.md:173]: The English version of the documentation currently on master https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/execution_mode/#event-time--watermarks still points to the `docs/dev/datastream/event-time/generating_watermarks` page. I think the current situation is the right page to point users to and therefore we shouldn't make this change.

File: docs/content.zh/docs/dev/datastream/execution_mode.md (hunk 1 above)
Comment (MartijnVisser) [reply docs/content.zh/docs/dev/datastream/execution_mode.md:204]: The English documentation doesn't have a link to `sum()` so I think it's fine to remove the link here

File: docs/content.zh/docs/dev/datastream/execution_mode.md [hunk 3]
```diff
@@ -201,11 +201,11 @@ Checkpointing 用于故障恢复的特点之一是，在发生故障时，Flink
 
 `批`模式下的行为变化：
 
-* “滚动"操作，如 [reduce()]({{< ref "docs/dev/datastream/operators/overview" >}}#reduce) 或 [sum()]({{< ref "docs/dev/datastream/operators/overview" >}}#aggregations)，会对`流`模式下每一条新记录发出增量更新。在`批`模式下，这些操作不是"滚动”。它们只发出最终结果。
+* “滚动"操作，如 [reduce()]({{< ref "docs/dev/datastream/operators/overview" >}}#reduce) 或 sum()，会对`流`模式下每一条新记录发出增量更新。在`批`模式下，这些操作不是"滚动”。它们只发出最终结果。
 
 `批`模式下不支持的:
 
-* [Checkpointing]({{< ref "docs/concepts/stateful-stream-processing" >}}#stateful-stream-processing) 和任何依赖于 checkpointing 的操作都不支持。
+* [Checkpointing]({{< ref "docs/concepts/stateful-stream-processing" >}}#checkpointing) 和任何依赖于 checkpointing 的操作都不支持。
```
Comment (MartijnVisser) [inline docs/content.zh/docs/dev/datastream/execution_mode.md:208]: Good catch 👍

File: docs/content.zh/docs/dev/datastream/operators/overview.md [hunk 4]
```diff
@@ -680,6 +680,8 @@ data_stream.broadcast()
 {{< /tab >}}
 {{< /tabs>}}
 
+<a name="task-chaining-and-resource-groups"></a>
+
```
Comment (MartijnVisser) [inline docs/content.zh/docs/dev/datastream/operators/overview.md:684]: This is now missing the last pieces which are on https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/operators/overview/#name-and-description

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
Thanks for the PR @RocMarshal - I do think that there are some changes which still need to be fixed before we can merge this

File: docs/content.zh/docs/dev/datastream/operators/overview.md (hunk 4 above)
Comment (RocMarshal) [reply docs/content.zh/docs/dev/datastream/operators/overview.md:684]: @MartijnVisser Yes. I  executed the 'git rebase master'  command to resolve it.

**Review by RocMarshal — COMMENTED** [review]
@MartijnVisser @wuchong Thanks for your review. 
I made the change based on your opinions. 
PTAL. Thank you very much.

**Review by MartijnVisser — APPROVED** [review]
@RocMarshal I've rebased and squashed commits, I'll merge it when the CI has passed.

**Comment by MartijnVisser** [comment]
@RocMarshal Thanks again for the PR. Can you also create backports to the release-1.15 and release-1.14 branches?

**Comment by RocMarshal** [comment]
> @RocMarshal Thanks again for the PR. Can you also create backports to the release-1.15 and release-1.14 branches?

@MartijnVisser Thanks for the operation. I'm glad to do it and I'll create it asap.

**Comment by RocMarshal** [comment]
@MartijnVisser I created [BP-1.14](https://github.com/apache/flink/pull/19090) & [BP-1.15](https://github.com/apache/flink/pull/19091). 
PTAL. Thank you very much.

**Pull request merged**